_rules/audio-as-media-alternative-afb423.md

```diff
@@ -39,7 +40,7 @@ The label (from expectation 2) is [visible][] and [included in the accessibility
 
 ## Assumptions
 
-_There are currently no assumptions_
+This rule assumes that if `lang` attributes are used, their value accurately describes the part of the content they are applied to.
```

Comment:
@WilcoFiers this is an attempt to fulfil https://github.com/act-rules/act-rules.github.io/issues/671#issuecomment-523818574

Essentially, there is no way to automagically find the language of the page and the rule evaluator (whether a tool or a human) needs a way to know it.

Would it be better to add to each example description a sentence like "In this example, the language of the page is provided by the `lang` attribute"?

---
I think the question rises with an example like the one in 
```html
<html lang="en">
<p>Opskrifter på desserter:</p>
<ul>
<li><a href="ice-cream.html">Is</a></li>
</ul>
</html>
```
Is this passing "link name is descriptive"?
If we consider that the `lang` attribute is OK, then no, it is not. "is" is **not** descriptive of ice cream in English. OTOH, if we consider that the page is in Danish and the `lang` attribute is not correct, then this "is" is descriptive of ice cream and this pass "link is descriptive".

When evaluating the rule, how can I know whether the `lang` attribute is incorrect or whether the `p` element is incorrect and should be "recipes for desserts"?

Reply:
Would an assumption in the line of "This rule assume that the programatically determined language accurately reflects the actual language used" be good?

(I think it's at least better since it doesn't rely on `lang` attribute only)

Reply:
@Jym77 A language model is input to the rule. If you have a functional language model, it will tell you what the language of a text is, and what a text means. You don't have to make any assumptions, given the same language model as input, the results will be the same. 

We don't really need to add lang to our test cases. The reason we wanted to do it, is because it increases the likeliness that tools will run the test case in the same way. For shorter sections, which most test cases are, tools need a little help identifying language. Human evaluators don't really need this. Adding in a description isn't beneficial in my opinion.

Reply:
This comment hasn't been resolved. My review should not have been dismissed.

Reply:
The comment was answered and the answer was waiting for agreement or disagreement on your side. Which took one week to come. With several requests for it in the bi-weekly WAI-tools standups or in Slack.

Therefore, I dismissed the review, considering that the concern was answered. Same as what you did in https://github.com/act-rules/act-rules.github.io/pull/986#issuecomment-546935015 when you disagreed when my concern and simply dismissed my review.
And you are of course welcome to disagree with my answer and re-request the same changes, as you just did.

(and in the meantime we had a call and agreed to disagree and call on opinions of wider audience)